browser/resources/settings/email_aliases_page/content/remote_mapping_service.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { Alias, MappingService } from './types'
+
+export class RemoteMappingService implements MappingService {
```

Comment:
why is this called RemoteMappingService?

Reply:
Because the mapping service is on AWS.

Reply:
but why is that important for the UI? This should probably just be MappingServiceImpl?

Reply:
I just made a similar comment. MappingService is too generic for something browser-wide. And I agree RemoteMappingService doesn't make sense. This also wouldn't exist when mojom definitions are done properly as it would generate MappingServiceRemote, although it should probably be something like EmailAliasService and EmailAliasServiceRemote, right?

Reply:
These are now in mojom, and are called `EmailAliasesServiceInterface` etc.